crates/astria-proto/proto/astria/primitive/v1alpha1/types.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
```

Comment:
Should the primitives be versioned? These are workarounds for missing protobuf primitives and shouldn't change.

Idk which way I lean. I'm also fine with copying these into new versions at the cost of code duplication (the test, specifically).